snap/local/ros_network.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/bash
+
+ROS_MASTER_HOST="$(snapctl get ros-master-host)"
+
+if [[ -z "${ROS_MASTER_URI}" ]]; then
+    export ROS_MASTER_URI=http://${ROS_MASTER_HOST}:11311
+fi
+export ROS_IP=${ROS_MASTER_HOST}
```

Comment:
You should consider using `ROS_HOSTNAME` instead of `ROS_IP`. From the [doc](http://wiki.ros.org/ROS/EnvironmentVariables#ROS_IP.2FROS_HOSTNAME) *"Use ROS_IP if you are specifying an IP address, and ROS_HOSTNAME if you are specifying a host name."*